environments/production/app-processes.yml

```diff
@@ -72,6 +72,8 @@ celery_processes:
       prefetch_multiplier: 1
     ush_background_tasks:
       concurrency: 8
+    soltech_custom_script_queue:  # TODO: Remove after executing script
+    concurrency: 4
```

Comment:
indentation is off here?

Reply:
so this will add the queue to 4 machines. That is what we want?

Reply:
Ah, my silly IDE auto-formatted that incorrectly. Fixed in [f33980f](https://github.com/dimagi/commcare-cloud/pull/6335/commits/f33980f665ccf85669371c84ccd2a4a775a4a90e)